src/pages/blog/xt24-fintech-conf.mdx

```diff
@@ -0,0 +1,41 @@
+---
+author: 'juxt'
+layout: '../../layouts/BlogPost.astro'
+title: 'Get Ready to Elevate Your Fintech Experience and Networking at XT24'
+description: 'Engineering leaders from global banks, fintech companies and hedge funds are meeting in Milton Keynes for XT24 Fintech Conf'
+category: 'conference'
+publishedDate: '2024-04-25'
+tags:
+  - fintech
+  - tech talks
+  - conference
+heroImage: 'xt24-og-image.jpg'
+---
+
+For over a decade, JUXT has been synonymous with first-class engineering and unforgettable conferences.
+
+This May 9th, we're taking it up a notch with [XT24](https://www.juxt.pro/xt24/) – a premier in-person event dedicated to innovation in Fintech.
```

Comment:
rather than 'take it up a notch'., which sets high expectations, can you tone it down a bit?
e.g. "this year's conference is dedicated to technical innovation in fintech" or similar.